### PR TITLE
ROC-4653 Send settlement rejection email to staff

### PR DESCRIPTION
**Pull request by tharacka**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-4653

### Change description ###

Notify staff via email when a settlement is rejected

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/offers/CCDStatementType.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 public enum CCDStatementType {
     OFFER("offer"),
     ACCEPTATION("acceptation"),
-    REJECTION("rejection"),
+    REJECTION("rejected"),
```
[the remark this answers is not in the document]
Comment (tharacka) [reply ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/offers/CCDStatementType.java]: this shouldn't have been in this PR

File: src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/SettlementAgreementRejectionStaffNotificationServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
+
+public class SettlementAgreementRejectionStaffNotificationServiceTest extends MockSpringTest {
+
+    @Autowired
+    private SettlementAgreementRejectedStaffNotificationService service;
+
+    @Captor
+    private ArgumentCaptor<String> senderArgument;
```
[the remark this answers is not in the document]
Comment (tharacka) [reply src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/SettlementAgreementRejectionStaffNotificationServiceTest.java]: I have gone with this as it seems to be used in most places in the code

[the remark this answers is not in the document]
Comment (andrewwa-kainos) [reply src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/SettlementAgreementRejectionStaffNotificationServiceTest.java]: _Tu quoque_ is not a reason for bad code. I'm pointing this out and tidying it up wherever else I see it too.

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/SettlementAgreementRejectionEmailContentProvider.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import uk.gov.hmcts.cmc.claimstore.stereotypes.EmailContentProvider;
+
+import java.util.Map;
+
+import static org.apache.commons.lang3.Validate.notEmpty;
+
+@Component
+public class SettlementAgreementRejectionEmailContentProvider implements EmailContentProvider<Map<String, Object>> {
+
+
```
[the remark this answers is not in the document]
Comment (andrewwa-kainos) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/SettlementAgreementRejectionEmailContentProvider.java]: Please don't resolve comments until they're actually fixed or responded to

[the remark this answers is not in the document]
Comment (tharacka) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/SettlementAgreementRejectionEmailContentProvider.java]: Sorry have problems with git

**Pull request merged**